SU-SWS/sulgryphon-nextjs
# SUL23-60 Style the interior page header and footer

# READY FOR REVIEW

# Summary
- Styled the interior page header
- styled the library global footer

# Steps to Test
1, Review the deploy preview at https://637e5090c20c8b0008022cd6--stanford-library.netlify.app/test-basic-page-content/paragraph-components/test-text-area-content
2. verify it closely resembles the [mockups](https://www.figma.com/file/Zsz3AI81HpVF5Vnuxp3gwC/Stanford-Libraries-Design-02?node-id=2735%3A151399&t=RNkrnygPTJ4kkB0b-0).


<a href="https://gitpod.io/#https://github.com/SU-SWS/sulgryphon-nextjs/pull/37"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
